internal/website/content/howto/blob/open-bucket.md

```diff
@@ -0,0 +1,328 @@
+// above.
+client, err := gcp.NewHTTPClient(
+    gcp.DefaultTransport(),
+    gcp.CredentialsTokenSource(creds))
+if err != nil {
+    return err
+}
+
+// Create a *blob.Bucket.
+bucket, err := gcsblob.OpenBucket(ctx, client, "my-bucket", nil)
+if err != nil {
+    return err
+}
+defer bucket.Close()
+```
+
+[GCP creds]: https://cloud.google.com/docs/authentication/production
+[`gcsblob.OpenBucket`]: https://godoc.org/gocloud.dev/blob/gcsblob#OpenBucket
+
+## Azure Storage
+
+Azure Storage URLs in the Go CDK allow you to identify [Azure Storage][] containers
+when opening a bucket with `blob.OpenBucket`. Go CDK uses the environment
+variables `AZURE_STORAGE_ACCOUNT`, `AZURE_STORAGE_KEY`, and
+`AZURE_STORAGE_SAS_TOKEN` to configure the credentials. `AZURE_STORAGE_ACCOUNT`
+is required, along with one of the other two.
+
+```go
+import (
+    "gocloud.dev/blob"
+    _ "gocloud.dev/blob/azureblob"
+)
+
+// ...
+
+bucket, err := blob.OpenBucket(ctx, "azblob://my-container")
+if err != nil {
+    return err
+}
+defer bucket.Close()
+```
+
+Full details about acceptable URLs can be found under the API reference for
+[`azureblob.URLOpener`][].
+
+[Azure Storage]: https://azure.microsoft.com/en-us/services/storage/
+[`azureblob.URLOpener`]: https://godoc.org/gocloud.dev/blob/azureblob#URLOpener
+
+### Azure Storage Constructor
+
+The [`azureblob.OpenBucket`][] constructor opens an Azure Storage container.
+`azureblob` operates on [Azure Storage Block Blobs][]. You must first create
+Azure Storage credentials and then create an Azure Storage pipeline before
+you can open a container.
+
+```go
+import (
+    "github.com/Azure/azure-storage-blob-go/azblob"
+    "gocloud.dev/blob/azureblob"
+)
+
+// ...
+
+const (
+    // Fill in with your Azure Storage Acount and Access Key.
+    accountName = azureblob.AccountName("my-account")
+    accountKey  = azureblob.AccountKey("my-account-key")
+    // Fill in with the storage container to access.
+    containerName = "mycontainer"
+)
+
+// Create a credentials object.
+credential, err := azureblob.NewCredential(accountName, accountKey)
+if err != nil {
+    return err
+}
+
+// Create a Pipeline, using whatever PipelineOptions you need.
+pipeline := azureblob.NewPipeline(credential, azblob.PipelineOptions{})
+
+// Create a *blob.Bucket.
+// The credential option is required if you're going to use blob.SignedURL.
+ctx := context.Background()
+bucket, err := azureblob.OpenBucket(ctx, pipeline, accountName, containerName,
+    &azureblob.Options{Credential: credential})
+if err != nil {
+    return err
+}
+defer bucket.Close()
+```
+
+[Azure Storage Block Blobs]: https://docs.microsoft.com/en-us/rest/api/storageservices/understanding-block-blobs--append-blobs--and-page-blobs#about-block-blobs
+[`azureblob.OpenBucket`]: https://godoc.org/gocloud.dev/blob/azureblob#OpenBucket
+
+## Local Storage
+
+The Go CDK provides bucket drivers for storing in-memory and on the local
```

Comment:
maybe "storing data in-memory ...."

otherwise it's not clear what is stored, esp. with "on the local filesystem"

Reply:
Done.